gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/client/AbstractRestAction.java

```diff
@@ -75,23 +63,20 @@ public String getPath() {
     }
 
     @Override
-    public List<RestParameter> getPathParams() {
-        return pathParams;
-    }
-
-    @Override
-    public List<RestParameter> getQueryParams() {
-        return queryParams;
-    }
+    public List<HttpParameter> getParameters(Type type) {
+        List<HttpParameter> filteredParams = new ArrayList<HttpParameter>();
+        for (HttpParameter parameter : parameters) {
+            if (parameter.getType() == type && parameter.getObject() != null) {
+                filteredParams.add(parameter);
+            }
+        }
 
-    @Override
-    public List<RestParameter> getFormParams() {
-        return formParams;
+        return filteredParams;
     }
 
     @Override
-    public List<RestParameter> getHeaderParams() {
-        return headerParams;
+    public boolean isSecured() {
+        return false;
```

Comment:
With RPC it's easy to have it enabled by default because we provide the server library. Here we don't and most server libraries don't have csrf protections available / enabled by default.